docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -686,6 +820,45 @@ The following table lists all the playlist update mode attributes the client can
 >
 > This Attributes support Tizen 4.0 and Higher for Mobile.
 
+## Media Controller Content Type Attributes
+
+The following table lists all the content type attributes the server can receive.
```

Comment:
Please insert a colon in the end.

The following table lists all the content type attributes the server can receive: